### PR TITLE
Updated Portfolio with Projects and About Myself

### PR DESCRIPTION
**Pull request by mershark**
This is the second phase of building my portfolio website. In this stage, I added to the landing page My Projects and About Myself.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,15 +26,133 @@ <h2 class="introh2">LET’S CONNECT</h2>
...
+      <p class="card1-paragraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+      <ul class="card1-ul2">
+          <li class="card1-html">HTML</li>
+          <li class="card1-css">CSS</li>
+          <li class="card1-javascript">JavaScript</li>
+      </ul>
+      <button class="card1-button">See project</button>
+    </section>
+</section>
+
```
Comment (DuaneDave) [inline index.html]: With regards to the work section, I recommend the following changes:

- The whole section should be wrapped inside one `section tag that is aligned using `grid` and the project cards themselves would be using a `div` as wrapper. This is important in order to make your code semantic and to make it easier to style when building the desktop version.
- The heading tag for the project card should not be an `h1` tag since this tag is mostly used to open up a website mostly at the hero section, hence I recommend using maybe an `h3` here instead of `h1` to follow best practices.
- For interactivity purposes, the button here and for other occurrences of a button throughout the website should have a hover and pressed/active states.

File: index.html [hunk 2]
```diff
@@ -26,15 +26,133 @@ <h2 class="introh2">LET’S CONNECT</h2>
...
+          <ul class="card2-ul1">
+              <li class="card2-canopy">CANOPY</li>
+              <li class="card2-back-end-dev">Back End Dev</li>
+              <li class="card2-2015">2015</li>
+          </ul>
+      <p class="card2-paragraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+      <ul class="card2-ul2">
+          <li class="card2-html">HTML</li>
+          <li class="card2-css">CSS</li>
+          <li class="card2-javascript">JavaScript</li>
```
Comment (DuaneDave) [inline index.html]: For this section, I recommend the following:

- To make the code well structured, consider not to wrap a `section` tag within another since it can cause some problems with the accessibility of your website and can also make it hard to follow by other devs
- The heading tag for the project card should not be an h1 tag since this tag is mostly used to open up a website mostly at the hero section, hence I recommend using maybe an h3 here instead of h1 to follow best practices.
- For interactivity purposes, the button here and for other occurrences of a button throughout the website should have a hover and pressed/active states.

File: index.html [hunk 3]
```diff
@@ -26,15 +26,133 @@ <h2 class="introh2">LET’S CONNECT</h2>
...
+      <div class="socials">
+        <ul>
+          <li class="ls"><a href="#" class="twitter-hp"><img src="./images/twitter.svg" alt="This is a twitter button"></a></li>
+          <li class="ls"><a href="#" class="linkedin-hp"><img src="./images/Linkedin icon.svg" alt="Thi is a linkedin button"></a></li>
+          <li class="ls"><a href="#" class="unknw1"><img src="./images/Vector.svg" alt=""></a></li>
+          <li class="ls"><a href="#" class="github-hp"><img src="./images/github.svg" alt="This is a github button"></a></li>
+          <li class="ls"><a href="#" class="unknw2"><img src="./images/Vector (1).svg" alt=""></a></li>
+        </ul>
+        <button class="get-my-resume-button">Get my resume</button>
+      </div>
```
Comment (DuaneDave) [inline index.html]: For this section of the website, I recommend the following:


- To make the code well structured, consider not to wrap a section tag within another since it can cause some problems with the accessibility of your website and can also make it hard to follow by other devs
- The heading tag for the project card should not be an h1 tag since this tag is mostly used to open up a website mostly at the hero section, hence I recommend using maybe an h3 here instead of h1 to follow best practices.
- Since the languages, skills and frameworks are a list, I think it'd be better to arrange the language part inside the `ul` in order to be semantically okay.
- The section should be aligned with grid and not flex here in order to be on same page with the requirements.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @mershark 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ The UI looks impressive

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @mershark , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
